src/files/filesProcessor.ts

```diff
@@ -69,7 +74,13 @@ export const determineReleaseStatus = async (
           DonorMolecularDataReleaseStatus.PARTIALLY_RELEASED;
         break;
 
+      // Intentionally fall through to default so we can handle 3 or more states. More than 3 states should never occur.
```

Comment:
this isn't correct here, this comment was because i removed the break statement from case 3 and moved the logic for this action into the default case.